tests/e2e/map/__mocks__/colormaps/collections.body.json

```diff
@@ -29,7 +29,7 @@
                             {
                                 "antarctic": false,
                                 "group": "overlays",
-                                "product": "GHRSST_L4_MUR_Sea_Ice_Concentration",
+                                "product": "GHRSST_Sea_Ice_Concentration",
```

Comment:
I could use some clarification as to why these values had to change in a few of the mock files

Reply:
Again, these were changed because gibs no longer has GHRSST_L4_MUR_Sea_Ice_Concentration as a colormap, they renamed it to GHRSST_Sea_Ice_Concentration